.toolversion

```diff
@@ -0,0 +1 @@
+2.3.3
```

Comment:
I think naming this `.cloc-version` would be nicer since we already have stuff like `.ruby-version` etc in other platform.